src/styles/blocks/nav.scss

```
@@ -0,0 +1,48 @@
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+    display: flex;
+    align-items: center;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+  
+  &__link {
+    position: relative;
+    color: #000;
+    text-decoration: none;
+    line-height: 14px;
```

```suggestion
    line-height: 14px;
```
I think it`s unnecessary